ionoscloud/provider.go

```diff
@@ -104,20 +107,23 @@ func Provider() *schema.Provider {
 			NatGatewayRuleResource:      dataSourceNatGatewayRule(),
 			NetworkLoadBalancerResource: dataSourceNetworkLoadBalancer(),
 			NetworkLoadBalancerForwardingRuleResource: dataSourceNetworkLoadBalancerForwardingRule(),
-			TemplateResource:      dataSourceTemplate(),
-			BackupUnitResource:    dataSourceBackupUnit(),
-			FirewallResource:      dataSourceFirewall(),
-			S3KeyResource:         dataSourceS3Key(),
-			GroupResource:         dataSourceGroup(),
-			UserResource:          dataSourceUser(),
-			IpBlockResource:       dataSourceIpBlock(),
-			VolumeResource:        dataSourceVolume(),
-			NicResource:           dataSourceNIC(),
-			ShareResource:         dataSourceShare(),
-			ResourceIpFailover:    dataSourceIpFailover(),
-			DBaaSClusterResource:  dataSourceDbaasPgSqlCluster(),
-			DBaaSVersionsResource: dataSourceDbaasPgSqlVersions(),
-			DBaaSBackupsResource:  dataSourceDbaasPgSqlBackups(),
+			TemplateResource:                dataSourceTemplate(),
+			BackupUnitResource:              dataSourceBackupUnit(),
+			FirewallResource:                dataSourceFirewall(),
+			S3KeyResource:                   dataSourceS3Key(),
+			GroupResource:                   dataSourceGroup(),
+			UserResource:                    dataSourceUser(),
+			IpBlockResource:                 dataSourceIpBlock(),
+			VolumeResource:                  dataSourceVolume(),
+			NicResource:                     dataSourceNIC(),
+			ShareResource:                   dataSourceShare(),
+			ResourceIpFailover:              dataSourceIpFailover(),
+			DBaaSClusterResource:            dataSourceDbaasPgSqlCluster(),
+			DBaaSVersionsResource:           dataSourceDbaasPgSqlVersions(),
+			DBaaSBackupsResource:            dataSourceDbaasPgSqlBackups(),
+			ApplicationLoadBalancerResource: dataSourceApplicationLoadBalancer(),
+			ApplicationLoadBalancerForwardingRuleResource: dataSourceApplicationLoadBalancerForwardingRule(),
```

Comment:
Maybe we can shorten the name to ALBForwardingRuleResource? Just a suggestion

Reply:
Just for the constant, right?

Reply:
as you wish, you can change the function name also. It's kind of long. Not the actual resource name though